misc/config_tools/data/ehl-crb-b/dp4.xml

```diff
@@ -0,0 +1,187 @@
+            <kern_mod desc="The tag for kernel image which act as multiboot module, it must exactly match the module tag in GRUB multiboot cmdline.">Zephyr_RawImage</kern_mod>
+            <ramdisk_mod desc="The tag for ramdisk image which act as multiboot module, it must exactly match the module tag in GRUB multiboot cmdline." />
+            <bootargs desc="Specify kernel boot arguments">reboot=acpi</bootargs>
+            <kern_load_addr desc="The loading address in host memory for the VM kernel">0x8000</kern_load_addr>
+            <kern_entry_addr desc="The entry address in host memory for the VM kernel">0x8000</kern_entry_addr>
+        </os_config>
+        <legacy_vuart id="0">
+            <type configurable="0" desc="vCOM1 type">VUART_LEGACY_PIO</type>
+            <base desc="vUART0 (A.K.A COM1) enabling switch. Enable by exposing its base address, disable by returning invalid base address.">COM1_BASE</base>
+            <irq configurable="0" desc="vCOM1 irq">COM1_IRQ</irq>
+        </legacy_vuart>
+        <legacy_vuart id="1">
+            <type configurable="0" desc="vCOM2 type">VUART_LEGACY_PIO</type>
+            <base desc="vUART1 (A.K.A COM2) enabling switch. Enable by exposing its base address, disable by returning invalid base address.">COM2_BASE</base>
+            <irq configurable="0" desc="vCOM2 irq">COM2_IRQ</irq>
+            <target_vm_id desc="COM2 is used for VM communications. When it is enabled, please specify which target VM that current VM connect to.">1</target_vm_id>
+            <target_uart_id configurable="0" desc="target vUART ID that vCOM2 connect to">1</target_uart_id>
+        </legacy_vuart>
+        <console_vuart id="0">
+            <base desc="Console vuart (PCI based) enabling switch. Enable by exposing its base address, disable by returning invalid base address.">INVALID_PCI_BASE</base>
+        </console_vuart>
+        <communication_vuart id="1">
+            <base desc="Communicatin vuart (PCI based) enabling switch. Enable by exposing its base address, disable by returning invalid base address.">INVALID_PCI_BASE</base>
+            <target_vm_id desc="This vuart is used for VM communications. When it is enabled, please specify which target VM that current VM connect to.">1</target_vm_id>
+            <target_uart_id desc="target vUART ID that this vuart connects to">1</target_uart_id>
+        </communication_vuart>
+        <pci_devs desc="pci devices list">
+            <pci_dev desc="pci device" />
+            <pci_dev desc="pci device" />
+        </pci_devs>
+        <mmio_resources desc="mmio devices list to passthrough">
+            <TPM2 desc="TPM2 device">n</TPM2>
+        </mmio_resources>
+    </vm>
+    <vm id="1">
+        <vm_type desc="Specify the VM type" readonly="true">PRE_RT_VM</vm_type>
+        <name desc="vm_name">ACRN PRE-LAUNCHED VM1</name>
+        <guest_flags desc="Select all applicable flags for the VM" multiselect="true">
+            <guest_flag></guest_flag>
```

Comment:
Do we not need `GUEST_FLAG_LAPIC_PASSTHROUGH` and `GUEST_FLAG_RT` here to make this a Real-Time VM?

Reply:
Thanks for the reminder. I will add GUEST_FLAG_LAPIC_PASSTHROUGH and GUEST_FLAG_RT here